### PR TITLE
Remove unused, conflicting dependencies

### PR DESCRIPTION
**Pull request by kylesuss**
Thanks for this example project. A few things I noticed when setting this up myself:
1. Mocha & Chai are included in the deps, though they are not used anywhere. Rather, Jasmine is used as the test framework.
2. Karma Babel preprocessor is included as a dep and the settings block is in your karma config file, but the preprocessor isn't actually used anywhere. It's a little confusing for someone who is also trying to get this configured w/ Babel. I think by removing the unused dep, it will be more obvious that the webpack preprocessor is a way to transform JS w/ Babel.
3. There is a `.setup.js` file that I believe is not needed for this example project. It looks like it should be used in combination w/ JSDom given [this documentation](https://github.com/airbnb/enzyme/blob/master/docs/guides/jsdom.md) which I don't believe is necessary here since Karma has both the Chrome & Phantom JS launchers (and JSDom isn't even in the `package.json` file.

There are actually more dependencies listed in the `package.json` file which aren't used in this repo though I know their use case _if_ there was actually a webpack config, so maybe it makes sense to also either remove those or add a webpack config which utilizes them.

So this PR cleans that up a bit. Let me know your thoughts.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,12 +15,10 @@
     "babel-preset-airbnb": "^1.0.1",
     "babelify": "^7.2.0",
```
Comment (alexanderchan) [inline package.json]: If we're taking this opportunity to cleanup deps:
- babelify
- browserify
- watchify
- karma-browserify 

are probably not being used also

**Comment by alexanderchan** [comment]
Do the tests still pass without the .setup.js?  I think the load of it is hidden away a bit in [karma.conf.js](https://github.com/lelandrichardson/enzyme-example-karma-webpack/blob/master/karma.conf.js#L8).  Perhaps we need simple travis build on this proj that run npm test =)

**Comment by kylesuss** [comment]
tests

Yep, sure do. Tried both PhantomJS & Chrome.

I'll remove the browserify deps as well.

**Comment by kylesuss** [comment]
Done. Not sure I have permissions to setup travis for this repo otherwise I would help.

**Comment by kylesuss** [comment]
Bump. Any additional feedback here?

**Comment by kylesuss** [comment]
I suppose this can be closed ;)

**Pull request closed without merging**